src/recommendations/track.js

```diff
@@ -17,13 +17,17 @@ export const createLinkTracker = (tracker, href, openInNewTab = false) => (e) =>
   return setTimeout(() => { global.location.href = href; }, LINK_TIMEOUT);
 };
 
-export const trackRecommendationsClicked = (courseKey, isControl, position, userId, href, recommendationType) => {
+export const trackRecommendationsClicked = (
```

Comment:
can we please rename it?
```suggestion
export const trackRecommendationClick = (
```